storage_manager_new.py

```diff
@@ -54,135 +58,122 @@ def show_all_items(self):
         self.model = model
-        Item.instances.append(self)
 
     @classmethod
-    def instance_from_string(cls, string):
-        (category, type, model) = string.split(', ')
-        cls(category, type, model)
+    def create_instance_from_string_and_write_to_database_file(cls, string):
+        instance = Item.create_instance_from_string(string)
+        instance.write_to_database_file()
 
     @classmethod
-    def create_instance_from_string_and_write_to_database_file(cls, string):
-        Item.instance_from_string(string)
+    def create_instance_from_string(cls, stringified_item):
+        (category, type, model) = stringified_item.split(', ')
+        return cls(category, type, model)
+
+    def write_to_database_file(self):
         File = open(Storage.database_file, 'a')
-        File.write(string + '\n')
+        File.write(f'{self.category}, {self.type}, {self.model}\n')
         File.close()
 
 
 class Interface:
-    def __init__(self, name, option_1, option_2, option_3, option_4, option_5):
-        self.name = name
-        self.option_1 = option_1
-        self.option_2 = option_2
-        self.option_3 = option_3
-        self.option_4 = option_4
-        self.option_5 = option_5
+    option_dict = {1: 'Add item', 2: 'Show all items',
+                   3: 'Show list of items from a certain category', 4: 'Delete item', 5: 'Delete all items'}
 
-    def menu_option_list(self):
-        option_list = [self.option_1, self.option_2,
-                       self.option_3, self.option_4, self.option_5]
-        return option_list
+    def __init__(self, name, storage):
+        self.storage = storage
+        self.start_program_interface(storage)
 
     def start_program_interface(self, storage):
-        print('What would you like to do? Type a number.\n')
-        self.show_action_options_and_execute_user_choice(storage)
+        while True:
+            print('What would you like to do? Type a number.\n')
+            self.show_option_menu_and_execute_user_choice_from_menu(storage)
+
+    def show_option_menu_and_execute_user_choice_from_menu(self, storage):
+        self.show_option_menu()
+        self.execute_user_choice_from_menu(storage)
+
+    def show_option_menu(self):
+        for number, action in self.option_dict.items():
+            print(f'{number}. {action}')
 
-    def show_action_options_and_execute_user_choice(self, storage):
-        menu = self.create_and_show_option_menu()
-        choice = Interface.get_user_choice(None)
+    def execute_user_choice_from_menu(self, storage):
+        choice = input()
         try:
             choice = int(choice)
-        except:
+        except ValueError:
             print('\nPlease pick number from the options printed above.\n')
             self.start_program_interface(storage)
-        if choice in menu.keys():
+        if choice in self.option_dict.keys():
             if choice == 1:
-                storage.add_item()
+                self.storage.add_item()
```

Comment:
As in one of previous comments - these should call `Interface` methods that will handle interaction with user and these methods will call storage ones only to do exactly what is in the name, like add or delete item(s).
In this way, all classes are doing what they should be doing - `Interface` class handles, well, interface and communication to user and `Storage` handles only database managing, not caring who is user and how we communicate with him.